sickbeard/providers/tvchaosuk.py

```diff
@@ -102,7 +100,7 @@ def search(self, search_strings, age=0, ep_obj=None):  # pylint: disable=too-man
             for search_string in search_strings[mode]:
 
                 if mode == 'Season':
-                    search_string = re.sub(ur'(.*)Season', ur'\1Series', search_string)
+                    search_string = re.sub(ur'(.*)S0?', ur'\1Series', search_string)
```

Comment:
What about seasons > 9? And series with a year for a season?
How about `ur'\bs\d{1,4}\b'` instead?  It will match any Sx through Sxxxx. Also since it doesn't have the `.* ` it will match multi-season too.